src/components/cveRecordSearchModule.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="field mt-1" style="">
+    <div class="field has-addons mb-1">
+      <div class="control is-expanded">
+        <input v-model.trim="queryString" @keyup.enter="onKeyUpEnter" @keyup="validateQueryString" type="text" class="input cve-id-input"
+          placeholder="Enter keywords (e.g.: CVE ID, sql injection, etc.)"/>
+      </div>
+      <div class="control">
+        <button @click="startSearch"
+          class="button cve-button cve-button-accent-warm"
+          :class="{'is-loading': cveListSearchStore.isSearching, 'disabled': cveListSearchStore.isSeachButtonDisabled}"
+          :aria-disabled="cveListSearchStore.isSeachButtonDisabled">
+          Search {{ websiteEnv === 'test' ? 'CVE List in Test' : ''}}
+        </button>
+      </div>
+    </div>
+    <div class="notification is-warning is-light" role="alert" v-if="errorMessage.length > 0">
+      <div class="is-flex" style="justify-content: flex-start;">
```

Comment:
I think you can use Bulma's `is-align-content-flex-start` class for this, instead of a separate class and style